vanilla-framework/vanilla-framework
# 1667: Hide text inside social icons

## Done

- Changed social icons so that text inside is hidden. Best practice would probably be to add `aria-label` to the element but this is a nice fallback.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-social/
- Add text inside any of the icon elements
- Check that nothing shows up

## Details

Fixes #1667
